shared/config/resources.go

```diff
@@ -24,15 +24,9 @@ type StakewiseResources struct {
 	// See https://github.com/stakewise/v3-core/blob/main/contracts/vaults/ethereum/mev/SharedMevEscrow.sol
 	FeeRecipient *common.Address
 
-	// The splitter proxy used by the NodeSet vault, which distributes rewards to each node op.
-	// All Beacon + EL rewards from the Vault and FeeRecipient will end up here at some point, but they will be in the ERC-20 represented by the Vault.
-	// Funds here can be sent to SplitMain by calling `Distribute` on SplitMain and passing this address in as part of it.
-	// TODO: Find the "share" per node op address
-	SplitWallet *common.Address
-
-	// The address of the SplitMain contract, see https://docs.splits.org/core/split for details.
+	// The address of the SplitWarehouse contract used to hold user funds.
```

Comment:
Probably worth putting a comment line back in here with a URL to the Splits docs site for our own reference later.

Reply:
Fixed